README.md

```diff
@@ -30,36 +31,12 @@ in the remote registry, right next to the image itself. At container launch time
 SOCI Snapshotter queries the registry for the presence of the SOCI index using the mechanism
 developed by the [OCI Reference Types working group](https://github.com/opencontainers/wg-reference-types).
 
-### Workload-specific load order optimization
-
-Some lazy loading snapshotters support load order optimization, where some files are
-prioritized for prefetching. Typically, there is a one-to-one relationship between
-the list of to-be-prefetched files and the image or layer artifact.
-
-For SOCI, we wanted a bit more flexibility. Often, which files to prefetch is highly
-dependent on the specific workload, not the image or base layer. For example, a customer
-may have a Python3 base layer that is shared by thousands of applications. To optimize
-the launch time of those applications using the traditional approach, the base
-layer can no longer be shared, because each application’s load order for that layer will be
-different. Registry storage costs will increase dramatically, and cache hit rates will plummet.
-And when it comes time to update that base layer, each and every copy will have to be reoptimized.
-
-Secondly, there are some workloads that need to be able to prefetch at the subfile level. For example,
-we have observed machine learning workloads that launch and then immediately read a small header
-from a very large number of very large files.
-
-To meet these use-cases, SOCI will implement a separate load order document (LOD), that can specify
-which files or file-segments to load. Because it is a separate artifact, a single image can have
-many LODs. At container launch time, the appropriate LOD can be retrieved using business logic
-specified by the administrator.
-
-***Note:*** **SOCI Load order optimization is not yet implemented in SOCI.**
+## Documentation
 
-## Learning More
-Check out our docs area:
-- [Getting
-Started](docs/GETTING_STARTED.md)
-- [Glossary](docs/GLOSSARY.md)
+- [Getting Started](docs/GETTING_STARTED.md): walk through SOCI setups and features.
+- [BUILD](docs/BUILD.md): how to build SOCI from source, test SOCI (and contribute).
+- [INSTALL](docs/INSTALL.md): how to install SOCI as a systemd unit.
+- [Glossary](docs/GLOSSARY.md): glossary we use in the project.
```

Comment:
Can we make capitalization here consistent?

Reply:
fixed